Assets/Scripts/Items/Oxygen.cs

```diff
@@ -1,14 +1,19 @@
-﻿using UnityEngine;
+﻿using Combat;
+using UnityEngine;
 
 namespace Items {
     [DisallowMultipleComponent]
     [RequireComponent(typeof(Interactable))]
     public class Oxygen : MonoBehaviour
     {
+        [SerializeField] public int healingValue = 10;
```

Comment:
This attribute is not needed since the field is already serialized.

```suggestion
        public int healingValue = 10;
```